google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionClientTests.java

```diff
@@ -143,13 +145,16 @@ public void batchCreateAndCloseSessions() {
     final String sessionName = dbName + "/sessions/s%d";
     final Map<String, String> labels = new HashMap<>();
     labels.put("env", "dev");
+      String databaseRole = new String("role");
```

Comment:
nit: indentation seems to be off here (and also elsewhere in this class). Maybe run code formatter again?